tests/test_beacon_network.py

```diff
@@ -124,3 +180,22 @@ def test_network_local_beacon_query(app_config, client, aioresponse):
     mock_network_init(app_config, aioresponse)
     response = client.post(f"/network/beacons/local.test.beacon/individuals", json=BEACON_REQUEST_BODY)
     assert response.status_code == 200
+
+
+
+
```

Comment:
I'd expect `black` to reduce this to a double-newline, but maybe the auto-formatter doesn't touch tests?

Reply:
still formatting by hand, and just forgot to do it. I guess lint check ignores tests.